### DISCUSSION TITLE
SyntaxError: Unexpected non-whitespace character after JSON at position 4

### DISCUSSION THREAD

**Discussion by atharvnuthi** [aiortc/aiortc#966] [Q&A]
Hey! Currently trying to implement something using [https://github.com/aiortc/aiortc/tree/main/examples/webcam].
And sometimes, out of nowhere, it generates this popup:

Sometimes when I rename a variable. 

![Captura de Tela (4)](https://user-images.githubusercontent.com/57318481/235240230-8651bcd6-a392-40f2-b2c7-644bea8c5292.png)

I believe it is because the .json format that is being sent is invalid, through Offer.

**Comment by achaosss** [comment]
same questions

**Comment by Zark1999** [comment]
I downgraded cryptography version to 38 to bypass this problem.  'pip install cryptography==38'
aiortc -- 1.5
cryptography -- 38

Hope it helps.

--- 

Find another wierd case that can cause the problem. If you are using codec like 'codecs.get_capabilities('video')' without importing it using 'from aiortc import codec',  same error might occur in the web.

One helpful way to find out the cause is to wrap your aiohttp api code with try-except.

**Comment by cacard** [comment]
same error.

windows + edge
